### PR TITLE
MAINT: Fix nightly by skipping and noting problematic code

### PR DESCRIPTION
**Pull request by bashtage**

### CODE REVIEW

File: tests/test_errors.py [hunk 1]
```diff
@@ -1,46 +1,71 @@
...
-    PossibleDataLossError,
-)
-from pandas.io.sql import DatabaseError
-from pandas.io.stata import (
-    CategoricalConversionWarning,
-    InvalidColumnName,
-    PossiblePrecisionLoss,
-    ValueLabelTypeMismatch,
-)
+PD_LT_15 = parse(pd.__version__) < parse("1.5.0")
```
Comment (Dr-Irv) [inline tests/test_errors.py:11]: Move `PD_LT_15` to `tests/__init__.py`, then you don't have to `from packaging.version import parse` in all those files.

File: tests/test_resampler.py [hunk 2]
```diff
@@ -297,7 +301,9 @@ def s2scalar(val: Series) -> float:
 
     check(S.resample("m").aggregate(np.sum), Series)
     check(S.resample("m").aggregate("sum"), Series)
-    check(S.resample("m").aggregate(s2series), Series)
+    if PD_LT_15:
+        # Warns on 1.5.0, add pytest.warns
```
Comment (Dr-Irv) [inline tests/test_resampler.py:305]: Couldn't you just do something like
```python
        if PD_LT_15:
            check(S.resample("m").aggregate(s2series), Series)
        else:
             with pytest.warns(FutureWarning, match=" FutureWarning: Not prepending group keys to the result index of transform-like apply"):
                 check(S.resample("m").aggregate(s2series), Series)
```

**Review by Dr-Irv — CHANGES_REQUESTED** [review]

File: tests/test_errors.py (hunk 1 above)
Comment (bashtage) [reply tests/test_errors.py]: Fixed.

File: tests/test_resampler.py (hunk 2 above)
Comment (bashtage) [reply tests/test_resampler.py]: Fixed.

**Comment by bashtage** [comment]
Should be good now.

**Review by Dr-Irv — APPROVED** [review]
thanks @bashtage

**Pull request merged**